frontend/src/slices/chats/chats.slice.ts

```diff
@@ -0,0 +1,54 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { DataStatus } from '#libs/enums/enums.js';
+import { type ValueOf } from '#libs/types/types.js';
+import { type ChatGetAllItemResponseDto } from '#packages/chats/chats.js';
+
+import { createChat, getAllChats } from './actions.js';
+
+type State = {
+  chats: ChatGetAllItemResponseDto[];
+  chatsDataStatus: ValueOf<typeof DataStatus>;
+  createChatDataStatus: ValueOf<typeof DataStatus>;
+};
```

Comment:
I see that, for example, for 'journal.slice', we name properties in such a way: '
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/aef09240-9c0c-478c-bd25-e0eed02d6373)
 (with the word 'entries' at the end. Perhaps we should use the same approach everywhere, what do you think?

Reply:
for journalEntries we name it like this because it's a name of our entity (journal_entry, like note), while journal is a feature name. Here chat is a name of our entity and a feature name as well